webapp/src/reducers.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines */
```

Comment:
Off topic but I am thinking your IDE is applying linting that's different from the one configured in repo. It would be nice to align at some point since otherwise I'd be constantly breaking more stuff for you.

Reply:
The repo settings are 
```
    "max-lines": [
      1,
      {
        "max": 450,
        "skipBlankLines": true,
        "skipComments": false
      }
    ],
```
So if we want to change that, we can, otherwise we're going to be getting that warning. 
But wrt to this one -- it's not breaking anything for me , I disabled it in this file because it was causing a lot of visual noise. :) But there are a ton of eslint warnings we're ignoring at the moment (like no `any`s). Maybe it's worth it to go through and clean up the warnings?

Reply:
I just think we should be consistent, not very opinionated about the details. If you are using different rules locally we should port them over to the repository so I can follow them as well. This is separate work of course, just bringing it up here since I've seen it before. A cleanup ticket would be nice.

Reply:
>  If you are using different rules locally we should port them over to the repository so I can follow them as well. 

I'm not using different rules locally, the repo settings I pasted are in the `.eslintrc.json` file in the repo.

Reply:
:thinking: Then we must be running a different version or something because neither the CI nor my local env are reporting an issue with that file.

Reply:
I guess it's the warning vs error maybe the real problem here?

Reply:
Yep, it's because it's a level 1 (warning).

Reply:
Okay, that makes sense now. Then let's plan to:

1. Review existing warnings, deciding if anything should be an error instead or not even a warning.
2. Fix remaining warnings and errors.

Reply:
https://mattermost.atlassian.net/browse/MM-49235